static/app/views/issueList/groupSearchViewTabs/draggableTabBar.tsx

```diff
@@ -188,9 +192,14 @@ export function DraggableTabBar({
                 setIsEditing={isEditing => setEditingTabKey(isEditing ? tab.key : null)}
                 onChange={newLabel => handleOnTabRenamed(newLabel.trim(), tab.key)}
               />
-              {tab.key !== 'temporary-tab' && tab.queryCount && (
+              {tab.key !== 'temporary-tab' && tab.queryCount !== undefined && (
                 <StyledBadge>
-                  <QueryCount hideParens count={tab.queryCount} max={1000} />
+                  <QueryCount
+                    hideParens
+                    hideIfEmpty={false}
```

Comment:
This was causing some weird behavior if queryCount was 0 - it would show the 0 but not within the badge, so the tab title would look like "Prioritized0". This change fixes that bug, and it also will display a 0 in the badge, though we can hide the 0 query count by setting `hideIfEmpty` to true in the future if we want